sunpy/timeseries/timeseriesbase.py

```diff
@@ -119,6 +119,16 @@ def __init__(self, data, meta=None, units=None, **kwargs):
 
 # #### Attribute definitions #### #
 
+    @property
+    def data(self):
+        warnings.warn("use `.to_dataframe` instead of `.data` to get the timeseries as a pandas dataframe.",
```

Comment:
I think something more like 'Using `.data` to access the dataframe is discouraged; use `.to_dataframe()` instead'